docs/src/terminology.md

```diff
@@ -81,7 +90,11 @@ The wallclock duration between a [leader](#leader) creating a [tick entry](#tick
 
 ## confirmed block
 
-A [block](#block) that has received a [super majority](#supermajority) of [ledger votes](#ledger-vote).
+A [block](#block) for which a [super majority](#supermajority) of
+stake-weighted [votes](#vote) have been observed. Cannot be rolled
+back given < 5% defective stake.
```

Comment:
"defective" isn't currently a term we really use for stake, I'm worried this'll just cause different confusion.

Perhaps this: This block cannot be rolled back as long as less than 5% of the current stake weight is dishonest

Reply:
i don't find the terms that over-specify intention to be particularly useful. a buggy implementation is just as problematic as one deliberately designed to do harm in these scenarios. we've used "non-conforming" in the past, but maybe not in docs